src/standalone/userJupyterServer/serverSelectorForTests.ts

```diff
@@ -30,7 +30,7 @@ export class JupyterServerSelectorCommand
     ) {
         super();
     }
-    public readonly id = 'JupyterServerSelectorForTesting';
+    public readonly id = '_builtin.JupyterServerSelectorForTesting';
```

Comment:
Else we try to determine the extension that owns this provider, 
internally we use `_builtin.` prefix to indicate they are owned by Jupyter extension.